docs/user/reference/metrics/event.md

```diff
@@ -458,11 +457,13 @@ Each extra key contains additional metadata:
 
 ## Limits
 
-* When 500 events are queued on the client an events ping is immediately sent.
-* The `extra_keys` allows for a maximum of 10 keys.
+* When 500 events are queued on the client an "events" ping is immediately sent.
+* When 500 events are queues on the client for a Custom Ping,
+  subsequent events are dropped and errors of type `invalid_overflow` are recorded.
```

Comment:
It's [the designed behaviour](https://docs.google.com/document/d/1R-tIZYz4Izx8rDB_YUiJXU9Xtk8VcJoF8Pmhggps_GU/edit#heading=h.2wb70llunlqy). Let me check the code... You know [I think you're right](https://github.com/mozilla/glean.js/blob/main/glean/src/core/metrics/events_database/index.ts#L170-L206).

So, uh, which would you prefer? I like "don't enforce the upper limit" as it has less of a  "Surprise! Data Loss!" vibe.

Reply:
Huh.
I'm kinda surprised it's the designed behavior. 
The Glean.js implementation is thus basically a spec violation.

But IMO it's the better decision. The whole "rethink threshold" thing was to explicitly have custom pings not need to adhere to that, because they ultimately should not track how many events they added anyway.
I don't remember why we specified the overflow behavior.